### ISSUE TITLE
Unusable RFTools syringes listed in JEI

### ISSUE THREAD

**Issue by Brettflan** [DarkPacks/SevTech-Ages#2289]
## Issue / Bug
In JEI, full syringes for seemingly every mob added by various mods are listed, which implies (to me, anyway) that they work. However, only the default mobs and (as of 3.0.7) wild dogs from Animalium are actually supported in the Spawner, as it requires additional config for every supported mob.

## Possible Solution
Removing/hiding all of the other various still unsupported full RFTools syringes listed in JEI. Possibly even adding support for some of those mobs in the Spawner as well. Just off the top of my head, being able to spawn Lesser Shoggoths (at high cost) might be cool.

## Context
By default, only the empty RFTools syringe is listed in JEI, where this pack shows all of the full syringes as well (possibly due to staging?).

**Comment by artdude543** [comment]
Okay so after spending over 2 hours on this issue this is my conclusion.

1. Due to how the staging mod works and how RFTools is making the "items" it will create them as separate views in JEI. (An after effect from staging)
2. We can not use JEI (Config) to blacklist all the items. As technically it's only one item and all the ones listed are just "ghosts" from CraftTweaker/Staging which is creating the ItemStack.
3. I've tried staging the NBT versions of items but again the staging order (from the mods) differs on load. And items are staged all over the place.

So TLDR: This is not an easy thing to fix w/o some hacking which I don't feel is valid for this issue. What we could do is add a tooltip to show that not all them work/have a use. If you feel that would be fine @Brettflan but in terms of what we can do right now. There is not a whole lot.

So on that basic, going to close the issue. But I have this on internal notes to re-look when we can to see about other options/routes we could take.
